packages/suite/src/components/onboarding/UnexpectedState/index.tsx

```diff
@@ -6,23 +6,23 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
 import * as STEP from '@suite/constants/onboarding/steps';
-import PinMatrix from '@suite/components/onboarding/PinMatrix';
-import Text from '@suite/components/onboarding/Text';
 import { ConnectReducer } from '@suite/types/onboarding/connect';
 import { AnyStepDisallowedState } from '@suite/types/onboarding/steps';
 import { getFeatures, submitNewPin } from '@onboarding-actions/connectActions';
+import { PinMatrix, Text, Wrapper } from '@onboarding-components';
 import { Dispatch } from '@suite-types';
 import l10nMessages from './index.messages';
 import Reconnect from './Reconnect';
-import { ControlsWrapper } from '../Wrapper';
 
-const Wrapper = styled.div`
+const CommonWrapper = styled.div`
     margin: auto 30px auto 30px;
     text-align: center;
     width: 100%;
 `;
 
-const UnexpectedStateCommon: React.SFC = ({ children }) => <Wrapper>{children}</Wrapper>;
+const UnexpectedStateCommon: React.SFC = ({ children }) => (
```

Comment:
I don't like that React.SFC type, why not just use plain function with props?
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components
https://medium.com/@martin_hotell/10-typescript-pro-tips-patterns-with-or-without-react-5799488d6680 (last tip)

Reply:
:D I like it exactly for its simplicity - that I dont need to define children manually, because I usually recall proper type for it. 

But it seems that you, (and the tips) are right, for example this one sounds reasonable

> "FC defines optional children on props, which is not what your API may support as explained in tip no 8. API should be explicit!"